chrome/content/zotero/xpcom/data/item.js

```diff
@@ -3740,7 +3740,7 @@ Zotero.Item.prototype.getBestAttachmentState = async function () {
 	else {
 		type = 'other';
 	}
-	var exists = await item.fileExists();
+	var exists = !item.isFileAttachment() || await item.fileExists();
```

Comment:
Wouldn't a better fix just be to change `_canGetBestAttachmentState()` to return false for non-file attachments?

Reply:
Sure, makes sense.